### ISSUE TITLE
Alerts not being updated (but recreated)

### ISSUE THREAD

**Issue by marcfon** [timelyart/Kairos#53]
Could you provide some clarification on the documentation below?

```    
# The name is used to identify already existing alerts on your alert list in TradingView and is used to purge/update previously added alerts 
# Make sure the name is unique
```

When I try this, alerts with the same name don't get updated by re-created and added to the existing list of alerts.

Note that I do have multiple alerts that have the same name but act on different pairs.

**Comment by timelyart** [comment]
They get created a 2nd time? Instead of replacing the old one?

**Comment by marcfon** [comment]
> They get created a 2nd time? Instead of replacing the old one?

Yes, exactly that. How does the matching currently work? I have multiple alerts with the same name but configured for different pairs.

**Comment by timelyart** [comment]
My apologies for not responding earlier. 

It works by deleting the existing alerts prior to running the process. I guess that part broke down.

**Comment by timelyart** [comment]
Fixed in [release v2.83](https://github.com/timelyart/Kairos/releases/tag/v2.83).

Note that alerts are deleted and then re-created and not edited but the effect is the same.
